com.unity.render-pipelines.universal/Editor/ShaderGUI/ShadingModels/LitGUI.cs

```diff
@@ -129,14 +129,16 @@ public static void Inputs(LitProperties properties, MaterialEditor materialEdito
 
             // Check that we have all the required properties for clear coat,
             // otherwise we will get null ref exception from MaterialEditor GUI helpers.
-            if (   material.HasProperty("_ClearCoat")
-                && material.HasProperty("_ClearCoatMap")
-                && material.HasProperty("_ClearCoatMask"))
-                //&& material.HasProperty("_ClearCoatSmoothness")) //TODO: enable
-            {
-                //DoClearCoat(properties, materialEditor, material);
-            }
+            if ( ClearCoatAvailable(material) )
+                DoClearCoat(properties, materialEditor, material);
+        }
 
+        private static bool ClearCoatAvailable(Material material)
+        {
+            return    material.HasProperty("_ClearCoat")
+                   && material.HasProperty("_ClearCoatMap")
+                   && material.HasProperty("_ClearCoatMask")
+                   && material.HasProperty("_ClearCoatSmoothness");
```

Comment:
shouldn't this be an ||, I mean can't we have clear coat without having a clear coat map?

Reply:
Technically yes, but that would introduce inconsistency. We could have clear coats with different inputs.
This is currently only used to ensure all the properties required for the UI exist. 
_ClearCoatMap without Mask & Smoothness isn't a valid combination at the moment (but it could be), but the Mask & Smoothness without a Map is, so the condition would be (Map & Mask & Smoothness) | (Mask & Smoothness).
If we change this then we would need to do finer grain checks in the UI code, so that it can't fail.